ansible/custom_roles/bamboo/defaults/main.yml

```diff
@@ -17,7 +17,7 @@ bamboo_context_path: ~
 bamboo_home: "/var/lib/bamboo"
 bamboo_jvm_maximum_memory: "1024m"
 bamboo_jvm_minimum_memory: "512m"
-bamboo_jvm_support_recommended_args: "-Datlassian.plugins.enable.wait=300"
+bamboo_jvm_support_recommended_args: "-Datlassian.plugins.enable.wait=300 -Djava.net.preferIPv4Stack=true -Djava.net.preferIPv4Addresses=true"
```

Comment:
This will be applied to production too when we apply this ansible, but it should be alright I reckon.

Reply:
Yeah, so bamboo was just listening on IPv6 which is why the URL was unreachable for me. This just enables bamboo to listen on IPv4 rather than IPv6.